astro/src/content/docs/apis/_application-response-body-base.mdx

```diff
@@ -96,26 +96,40 @@ import Xmlsignaturec14nmethodValues from 'src/content/docs/_shared/_xmlSignature
     * `SlidingWindow` - the expiration is calculated from the last time the token was used.
```

Comment:
You need to add this to astro/src/content/docs/apis/_application-request-body.mdx also.